lib/bigcommerce/connection.rb

```diff
@@ -125,6 +129,7 @@ def request_legacy(method, path, options, headers={})
       resource_options = {
           :user => @configuration[:username],
           :password => @configuration[:api_key],
+          :verify_ssl => @configuration[:verify_ssl] || OpenSSL::SSL::VERIFY_PEER,
```

Comment:
Is this needed? The default for rest client is to enable peer, and if you want to disable it, you can just use the `:verify_peer`?